modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Details.tsx

```diff
@@ -138,6 +139,68 @@ const Details = ({
 				</ClayForm.Group>
 			</ClayLayout.SheetSection>
 
+			<ClayLayout.SheetSection className="mb-4">
+				<h3 className="sheet-subtitle">
+					{Liferay.Language.get('rest-information')}
+				</h3>
+
+				<ClayList className="flex-row flex-wrap">
```

Comment:
ClayList seems a little odd here, I am assuming it is for this bit?
![Screenshot 2023-06-12 at 11 17 48 AM](https://github.com/liferay-frontend/liferay-portal/assets/6843530/b73e8092-d539-4f08-a3a3-195cb3290b1d)


I think you could use ClayLayout for this instead, and that way you wouldn't need all the additional classnames for borders and col-*

Reply:
At first we used ClayLayout, but we asked in peds channel and they suggested to use Claylist to implement this design. I also asked for the best way to add responsiveness and they pointed me to that way of using classnames.

I personally think that it's better to use ClayList because it makes more sense semantically to use a list here, although it make us use extra classnames. But obviously I'm the new guy here :smile: